WebJobs.Script.sln

```diff
@@ -160,6 +160,49 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "BlobTrigger-Batch", "BlobTr
 		sample\BlobTrigger-Batch\run.bat = sample\BlobTrigger-Batch\run.bat
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "EasyTables", "EasyTables", "{3EACC3F3-2747-42D6-B826-6A150F9EA92A}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "CSharp", "CSharp", "{9A295BFF-005D-456B-BBEA-6F4F389B7A66}"
+	ProjectSection(SolutionItems) = preProject
+		sample\EasyTables\CSharp\host.json = sample\EasyTables\CSharp\host.json
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "QueueToOutItem", "QueueToOutItem", "{5DF1393D-B53A-47C3-895D-A7F3071F44F1}"
+	ProjectSection(SolutionItems) = preProject
+		sample\EasyTables\CSharp\QueueToOutItem\function.json = sample\EasyTables\CSharp\QueueToOutItem\function.json
+		sample\EasyTables\CSharp\QueueToOutItem\run.csx = sample\EasyTables\CSharp\QueueToOutItem\run.csx
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "QueueToInItem", "QueueToInItem", "{56AFFB2C-E153-48DD-B29F-F52842DDD1D5}"
+	ProjectSection(SolutionItems) = preProject
+		sample\EasyTables\CSharp\QueueToInItem\function.json = sample\EasyTables\CSharp\QueueToInItem\function.json
```

Comment:
See my other comment -- i should have said to ignore this. It works b/c I told the runtime to read from this folder rather than the root -- hence the new host.json. Worked like a charm :-)

---
In reply to: [56186335](https://github.com/Azure/azure-webjobs-sdk-script/pull/98#discussion_r56186335) [](ancestors = 56186335)